setup.py

```diff
@@ -26,12 +26,13 @@
         "absl-py>=0.9.0",
         "dataclasses>=0.8; python_version < '3.7'",
         "fonttools[ufo]>=4.17.0",
-        "importlib_resources>=3.3.0; python_version < '3.9'",
+        "importlib_resources>=3.3.0",
```

Comment:
Not sure why but importlib.resources on 3.9 isn't working for me, it says there are no resources

Reply:
let's keep like that until we investigate why built-in importlib.resources isn't working like the backport

Reply:
I am not sure why, but it looks like the built-in `importlib.resources` does work if we add an empty `__init__.py` module inside `src/nanoemoji/data` directory, next to the packaged default config file. Maybe it doesn't support "namespace" packages (i.e. without explicit `__init__.py`?). It may be worth adding the empty init file to make data an actual package, so we can use the built-in importlib.resources on python 3.9.

Reply:
yes, support for namespace packages looks like a recent thing only added to the `importlib_metadata` but not to the built-in one, see https://github.com/python/importlib_resources/pull/196
We should add `__init__.py` to data dir so that it is a normal, non-namespaced package. Ours just happened to be a "namespace", it wasn't intentional.

Reply:
I'm going to migrate that to a separate issue for now

Reply:
#198